frontend/public/pages/about.html

```diff
@@ -0,0 +1,14 @@
+<div id="about-container">
+    <h1 class="gradient-peach-to-maroon gradient-text font-poppins-regular">About Transcendence 42</h1>
+    <div>
+        <p class="text-color-text font-poppins-regular">Transcendence is a modern web application where fun meets functionality, a social gaming platform designed to bring players together through timeless classics.</p>
+        <p class="text-color-text font-poppins-regular">Challenge your friends or test your skills against others in Pong and Tetris, reimagined for the web with smooth real-time gameplay, dynamic leaderboards, and sleek design.</p>
+        <p class="text-color-text font-poppins-regular">Built as a full-stack end-of-cursus project by
+            <a href="https://profile-v3.intra.42.fr/users/ele-lean" class="author-link">ele-lean</a>,
+            <a href="https://profile-v3.intra.42.fr/users/etaquet" class="author-link">etaquet</a>,
+            <a href="https://profile-v3.intra.42.fr/users/releroy" class="author-link">releroy</a>,
+            <a href="https://profile-v3.intra.42.fr/users/tgoumand" class="author-link">tgoumand</a>
+            and <a href="https://profile-v3.intra.42.fr/users/tle-goff" class="author-link">tle-goff</a>, Transcendence 42 combines modern technologies with nostalgic gameplay to deliver an immersive and competitive online experience.</p>
```

Comment:
suggestion(Fix): Use `target="_blank"` for external links

These links point to an external domain, so the **dynamicLoader.js** won't load them correctly. 
Opening them in a new tab ensures the user can access the external profiles without breaking site functionality.

```suggestion
            <a href="https://profile-v3.intra.42.fr/users/ele-lean" target="_blank" class="author-link">ele-lean</a>,
            <a href="https://profile-v3.intra.42.fr/users/etaquet" target="_blank" class="author-link">etaquet</a>,
            <a href="https://profile-v3.intra.42.fr/users/releroy" target="_blank" class="author-link">releroy</a>,
            <a href="https://profile-v3.intra.42.fr/users/tgoumand" target="_blank" class="author-link">tgoumand</a>
            and <a href="https://profile-v3.intra.42.fr/users/tle-goff" target="_blank" class="author-link">tle-goff</a>, Transcendence 42 combines modern technologies with nostalgic gameplay to deliver an immersive and competitive online experience.</p>
```